src/Processors/Formats/Impl/NativeORCBlockInputFormat.h

```diff
@@ -8,6 +8,7 @@
 #    include <Processors/Formats/ISchemaReader.h>
 #    include <Storages/MergeTree/KeyCondition.h>
 #    include <boost/algorithm/string.hpp>
+#    include <orc/MemoryPool.hh>
```

Comment:
At first I thought that since we're only using a `unique_ptr` for this, we could just forward-declare it to avoid including its header.
However, `OrcFile.hh` -> `Reader.hh` -> `Vector.hh` -> `MemoryPool.hh`, so no matter what we do the header will still be included :disappointed: